### DISCUSSION TITLE
Astro & Panda: Entry layers

### DISCUSSION THREAD

**Discussion by thejackshelton** [chakra-ui/panda#1150] [Q&A]
I'm using the installation guide below:
https://panda-css.com/docs/installation/astro

In step #5 it mentions the following.

> Configure the entry CSS with layers

> Add this code to an index.css file imported in the root component of your project.

> @layer reset, base, tokens, recipes, utilities;

I'm not sure if there's really any root component for an Astro project. I put it in global.css inside of the styled-system directory. Would that be alright?

Not being able to style things in random parts of my website due to scoped styles was a major pain point for me. For example, I would use a poctcss media query variable plugin, only for it to work within that one page (even with is:global). I had to end up removing the plugin.

**Comment by thejackshelton** [comment]
Another thing, how does panda handle vendor prefixes? Does it use something like stylis?

**Comment by anubra266** [comment]
@thejackshelton You might be looking for this
https://github.com/chakra-ui/panda/pull/1154

**Reply by thejackshelton** [reply]
Oh this is great. I'm currently working on a real world project with Astro, Panda, and Storyblok. I haven't started on a second page yet, and didn't realize that it didn't add the base styles previously.

Looking to upgrade to that asap!

**Comment by thejackshelton** [comment]
@anubra266 can I use autoprefixer with Astro?

**Reply by anubra266** [reply]
The astro plugin already adds autoprefixer

**Reply by thejackshelton** [reply]
The new one? Or the old one, because my styles are broken on iOS

**Reply by thejackshelton** [reply]
This is the website for reference, still very new but building it right now https://website-8u6.pages.dev

**Reply by anubra266** [reply]
The old one
